doc/api/events.md

```diff
@@ -383,6 +383,17 @@ Installing a listener using this symbol does not change the behavior once an
 `'error'` event is emitted, therefore the process will still crash if no
 regular `'error'` listener is installed.
 
+### `EventEmitter.setEventTargetMaxListeners(n[, ...eventTargets])`
```

Comment:
Any objection to changing this to ``` ## `EventEmitter.setEventTargetMaxListeners(n[, ...eventTargets])` ``` and moving this up to the section with all the other `events.foo` things?

Reply:
It's a static on the `EventEmitter` class and is grouped with the other related functions (like `EventEmitter.listenerCount()`, I'd rather keep it where it's at.